src/deploy.js

```diff
@@ -0,0 +1,23 @@
+require('./setup');
+
+const ether = require('./ether');
+const send = require('./send');
+
+const { ERC1820_REGISTRY_DEPLOY_TX } = require('./data');
+
+async function ERC1820Registry (from) {
+  // Read https://eips.ethereum.org/EIPS/eip-1820 for more information as to how the ERC1820 registry is deployed to
+  // ensure its address is the same on all chains.
+
+  // 0.08 Ether is needed to deploy the registry, and those funds need to be transferred to the account that will deploy
```

Comment:
Capitalization of `Ether` looks weird to me. Should be either `ETH` or `ether` IMO.
```suggestion
  // 0.08 ether is needed to deploy the registry, and those funds need to be transferred to the account that will deploy
```